Doc/howto/descriptor.rst

```diff
@@ -1560,20 +1562,20 @@ Python:
     class Object:
         'Simulate how object.__new__() allocates memory for __slots__'
 
-        def __new__(cls, *args):
+        def __new__(cls, *args, **kwargs):
             'Emulate object_new() in Objects/typeobject.c'
             inst = super().__new__(cls)
             if hasattr(cls, 'slot_names'):
                 empty_slots = [null] * len(cls.slot_names)
-                object.__setattr__(inst, '_slotvalues', empty_slots)
+                super(Object, inst).__setattr__('_slotvalues', empty_slots)
```

Comment:
I wonder why the code was not already using `super().__setattr__`, given that super *is* used three lines before.
Maybe the author really wanted `object.__setattr__`.

Reply:
Let's let this as-is. The two argument form of `super()` is distracting and doesn't add value to the example.

Reply:
Updated.